package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-ui",
-    "version": "1.27.7",
+    "version": "1.28.0",
```

Comment:
No need update version, it will be counted by conventional commits when you will release by release tool taoRelease.
Version update needed only in backports, because in this case we do release manually